Java-Majasdarbi/src/main/java/advanced/test/Exercise1/CreditCard.java

```diff
@@ -0,0 +1,26 @@
+package advanced.test.Exercise1;
+
+public class CreditCard extends Card {
+    public CreditCard(String number, String fullName, String CCV, int balance) {
+        super(number, fullName, CCV, balance);
+    }
+
+
+    @Override
+    void withdrawMoney(int amount) {
+        checkCCV();
```

Comment:
Pirmkārt tā CCV pārbaude nebija prasīta, bet otrkārt - nav nedaudz dīvaini katru reizi gan tiek papildināta/izņemta nauda tas tiek pārbaudīts? Loģiskāk būtu to pārbaudīt pie kartes veidošanas konstruktorā, kurā Tu jau it kā pārbaudi to CCV bet ne līdz galam. Daļu pārbaudes atstāji šajā metodē, kuru katru reizi sauc.